packages/larva-css/src/03-utilities/u-padding.common.inline.scss

```diff
@@ -55,6 +55,12 @@ $paddings_with_breakpoints: (
 	('t', $spacer-050, tablet ),
 
 	('tb', $spacer-050, desktop ),
+	('lr', $spacer-050, desktop ),
```

Comment:
Group these in order by type, please! e.g. all desktop in order first, then group `a`s together, `tb`s, etc – I don't have a specific order for those though

Reply:
Also...Larva should not provide _all_ of the padding/margins needed, only commonly used ones (this can esp. get bloated with breakpoints b/c they are so specific to the design). 

Do you think these are common enough rules to include in Larva, or would they better coming from a `basic-utility-generator` for project-level utilities?

Reply:
@laras126 i think the main ones, like spacer-0, 1, 2 are common enough, but maybe for only like all, and not every single one (t, b, l, r). I will clean it up to avoid bloat!